styles/App.css

```diff
@@ -10,6 +10,7 @@
   --darker-gray-color: #1a1a1a;
   --black-color: #000;
   --green-color: #28a745;
+  --light-orange-color:#fedf81;
```

Comment:
I'm not really sure if the light orange colour is a good fit, perhaps @jcc-kh can feedback during prototyping of testimonials in https://github.com/AdvisorySG/mentorship-page/issues/735?

Reply:
For now I have removed the background colour. I have used the standard brand colour for the call to action.